adding ROADMAP.md (#6)
austinfrey/p2p-community-infrastructure: ROADMAP.md

# vision

> The initial vison for the CILG was to create place where anyone could learn about what is needed to host their own p2p infrastructure. It's less about protocols, apps, and products and more about identifying the best practices for managing/running hardware in the home, in the community or in the cloud.

Reviewer: ```suggestion
The initial vision for the Community Infrastructure Learning Group (CILG) was to create place where anyone could learn about what is needed to host their own p2p infrastructure. It's less about protocols, apps, and products, and more focused on identifying best practices for managing software and hardware in the home, in the community, or in the cloud.
```

# strategy

> Identify one use case from the p2p world and provide a happy path to easily setup the infrastructure needed to support that use case. This will include scripts, documentation in many forms i.e. traditional docs, blog style guides, video walk-throughs, online meetups etc etc. (Initially this will focus on SSB Pub setup). Once the CILG has worked through this process with this narrow use case, start identifying other use cases and replicate the process.

Reviewer: ```suggestion
Identify one use case from the p2p world and provide a happy path to easily set up the infrastructure needed to support that use case. This will include scripts, documentation in many forms, i.e. traditional docs, blog style guides, video walk-throughs, online meetups, etc. (initially this will focus on setting up a SSB Pub). Once the CILG has worked through this process with this particular use case, we will start identifying other use cases and by replicating our learned process.
```

> 

Reviewer: ```suggestion
```

# short term
- initial ssb pub scripts w/ accompanying documentation
  - documentation should include explanations for all commands so a user can become familiar with what's happening on their machine

> - scripts and docs will exist as a git repo, github or somewhere else.

Reviewer: ```suggestion
- scripts and docs will exist as a git repo, on GitHub or somewhere else
```

- pub setup can be a one line command but could also be done line by line "by hand"

> - accompanying tutorial style walk through (written, blog style)

Reviewer: ```suggestion
- accompanying tutorial style walk-through (written, blog style)
```

- online gathering/meetups for walk-throughs QA

# mid term
- package scripts as a debian package
- publish docs as an actual web page, this web page will serve as a hub for future efforts as well
- video pub setup tutorial
- online gathering/meetups for walk-throughs QA

# long term

> - Identify additional use cases and replicate the process.

Reviewer: ```suggestion
- identify additional use cases and replicate the process
```

> - Meat Space workshops

Reviewer: ```suggestion
- meat Space workshops
```

- Node School style lessons

> - Collaborate with other groups with like minded visions/values

Reviewer: ```suggestion
- collaborate with other groups with like minded visions/values
```
